src/pages/iou/steps/MoneyRequestAmount.js

```diff
@@ -485,9 +487,9 @@ function MoneyRequestAmountPage(props) {
     );
 }
 
-MoneyRequestAmountPage.propTypes = propTypes;
-MoneyRequestAmountPage.defaultProps = defaultProps;
-MoneyRequestAmountPage.displayName = 'MoneyRequestAmountPage';
+MoneyRequestAmount.propTypes = propTypes;
+MoneyRequestAmount.defaultProps = defaultProps;
+MoneyRequestAmount.displayName = 'MoneyRequestAmount';
 
 export default compose(
```

Comment:
Also, you are passing the `report` prop from the parent, so there's no need to subscribe to the current report in Onyx.

Reply:
same with `iou`

Reply:
No longer passing props in from parent, so this is out-of-date